device.go

```diff
@@ -178,6 +179,11 @@ type URLField struct {
 }
 
 func (uf *URLField) SetURLBase(urlBase *url.URL) {
+	str := uf.Str
```

Comment:
`str` looks to be unused, perhaps it should be used on line 187?

Reply:
already fix